code/game/objects/items/explosives/mine.dm

```diff
@@ -72,47 +112,65 @@
 
 	user.visible_message(SPAN_NOTICE("[user] starts deploying [src]."), \
 		SPAN_NOTICE("You start deploying [src]."))
-	if(!do_after(user, 40, INTERRUPT_NO_NEEDHAND, BUSY_ICON_HOSTILE))
+	playsound(loc, 'sound/machines/click.ogg', 25, 1)
+	if(!do_after(user, deploy_time, INTERRUPT_NO_NEEDHAND, BUSY_ICON_HOSTILE))
 		user.visible_message(SPAN_NOTICE("[user] stops deploying [src]."), \
-			SPAN_NOTICE("You stop deploying \the [src]."))
+			SPAN_NOTICE("You stop deploying [src]."))
 		return
 
 	if(active)
 		return
 
 	if(check_for_obstacles(user))
 		return
-
-	user.visible_message(SPAN_NOTICE("[user] finishes deploying [src]."), \
+	if(needs_digging)
+		playsound(loc, 'sound/weapons/flipblade.ogg', 25, 1)
+		user.visible_message(SPAN_NOTICE("[user] pulls the pin on [src]."), \
+			SPAN_NOTICE("You pull the activation pin and prepare it to be buried."))
+		user.drop_inv_item_on_ground(src)
+		anchored = TRUE
+		return
+	else
+		user.visible_message(SPAN_NOTICE("[user] finishes deploying [src]."), \
 		SPAN_NOTICE("You finish deploying [src]."))
-
+		if(user)
+			user.drop_inv_item_on_ground(src)
+			setDir(user ? user.dir : dir)
 	deploy_mine(user)
 
 /obj/item/explosive/mine/proc/deploy_mine(mob/user)
 	if(!hard_iff_lock && user)
 		iff_signal = user.faction
-
 	cause_data = create_cause_data(initial(name), user)
 	anchored = TRUE
-	playsound(loc, 'sound/weapons/mine_armed.ogg', 25, 1)
-	if(user)
-		user.drop_inv_item_on_ground(src)
-	setDir(user ? user.dir : dir) //The direction it is planted in is the direction the user faces at that time
+	playsound(loc, 'sound/weapons/mine_armed.ogg', 25, 1) //The direction it is planted in is the direction the user faces at that time
```

Comment:
Comment in the wrong place

Reply:
Doesn't seem to be in VSC.

Reply:
What ? It's the comment from old setDir() line that is put on the playsound() instead.